File: leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/TestActivity.java
package com.squareup.leakcanary.tests;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.fragment.app.FragmentActivity;
import com.squareup.leakcanary.support.fragment.R;

public class TestActivity extends FragmentActivity {

  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_test);
  }
}

File: leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/TestActivity.kt
package com.squareup.leakcanary.tests

import android.os.Bundle
import androidx.fragment.app.FragmentActivity
import com.squareup.leakcanary.support.fragment.R

class TestActivity : FragmentActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_test)
  }
}
